src/jit/lsra.cpp

```diff
@@ -3563,6 +3563,39 @@ void LinearScan::buildRefPositionsForNode(GenTree*                  tree,
     }
 #endif // DEBUG
 
+    const bool isContainedNode = !info.isLocalDefUse && consume == 0 && produce == 0 && tree->canBeContained();
+    if (isContainedNode)
+    {
+        assert(info.internalIntCount == 0);
+        assert(info.internalFloatCount == 0);
+
+        // Contained nodes map to the concatenated lists of their operands.
+        LocationInfoList locationInfoList;
```

Comment:
SuperPMI just finished; there are no diffs due to this change. I also checked AMD64 Unix via crossgen.